lib/components/Select.js

```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+import PropTypes from 'prop-types';
```

Comment:
This dependency needs to be declared in the package.json, both in `devDependencies` so it's installed for development, and in `peerDependencies` so it's marked as required for external use.